### PR TITLE
Facebook oauth #8

### PR DESCRIPTION
**Pull request by Mo-og**

### CODE REVIEW

**Comment by Mo-og** [comment]
|File|Coverage [34.2%]|:green_apple:|
|:-|:-:|:-:|
|[ApplicationErrorException.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FApplicationErrorException.java)|100%|:green_apple:|
|[WebSecurityConfig.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FWebSecurityConfig.java)|100%|:green_apple:|
|[InterviewerService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|94.3%|:green_apple:|
|[InterviewPlanningApplication.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2FInterviewPlanningApplication.java)|75%|:green_apple:|
|[User.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FUser.java)|65.61%|:green_apple:|
|[UserService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FUserService.java)|62.65%|:green_apple:|
|[AuthController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FAuthController.java)|45.1%|:green_apple:|
|[JwtAuthenticationEntryPoint.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtAuthenticationEntryPoint.java)|37.5%|:green_apple:|
|[UserController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FUserController.java)|30.33%|:green_apple:|
|[JwtRequestFilter.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtRequestFilter.java)|14.44%|:green_apple:|
|[AuthService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FAuthService.java)|4.95%|:x:|
|[JwtTokenUtil.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/fe7c3e45cdaeb76eb51afaa98648049fdbe788df/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtTokenUtil.java)|4.93%|:x:|

|Total Project Coverage|56.46%|:green_apple:|
|:-|:-:|:-:|

**Comment by Mo-og** [comment]
|File|Coverage [33.84%]|:green_apple:|
|:-|:-:|:-:|
|[ApplicationErrorException.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FApplicationErrorException.java)|100%|:green_apple:|
|[WebSecurityConfig.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FWebSecurityConfig.java)|100%|:green_apple:|
|[InterviewerService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|94.3%|:green_apple:|
|[InterviewPlanningApplication.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2FInterviewPlanningApplication.java)|75%|:green_apple:|
|[User.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FUser.java)|65.61%|:green_apple:|
|[UserService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FUserService.java)|62.65%|:green_apple:|
|[AuthController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FAuthController.java)|45.1%|:green_apple:|
|[JwtAuthenticationEntryPoint.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtAuthenticationEntryPoint.java)|37.5%|:green_apple:|
|[UserController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FUserController.java)|27.66%|:green_apple:|
|[JwtRequestFilter.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtRequestFilter.java)|13.27%|:green_apple:|
|[AuthService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FAuthService.java)|4.95%|:x:|
|[JwtTokenUtil.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/023fd2842f9a13758c855567d39af73f2758a146/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtTokenUtil.java)|4.93%|:x:|

|Total Project Coverage|55.21%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/models/User.java [hunk 1]
```diff
@@ -25,18 +35,23 @@
 @Setter
 @ToString
 @RequiredArgsConstructor
-@Table(name = "COORDINATOR")
+@Table(name = "USERS")
 @Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class User {
+@JsonInclude(Include.NON_NULL)
+public class User implements UserDetails {
```
Comment (Ewentualnie) [inline src/main/java/com/intellias/intellistart/interviewplanning/models/User.java:43]: Include.NON_NULL  maybe Include.NOT_NULL?

**Comment by Mo-og** [comment]
|File|Coverage [35.55%]|:green_apple:|
|:-|:-:|:-:|
|[ApplicationErrorException.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FApplicationErrorException.java)|100%|:green_apple:|
|[WebSecurityConfig.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FWebSecurityConfig.java)|100%|:green_apple:|
|[InterviewerService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|94.3%|:green_apple:|
|[InterviewPlanningApplication.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2FInterviewPlanningApplication.java)|75%|:green_apple:|
|[UserController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FUserController.java)|66.33%|:green_apple:|
|[User.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fmodels%2FUser.java)|65.61%|:green_apple:|
|[UserService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FUserService.java)|62.65%|:green_apple:|
|[AuthController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FAuthController.java)|45.1%|:green_apple:|
|[JwtAuthenticationEntryPoint.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtAuthenticationEntryPoint.java)|37.5%|:green_apple:|
|[JwtRequestFilter.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtRequestFilter.java)|13.27%|:green_apple:|
|[AuthService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FAuthService.java)|4.95%|:x:|
|[JwtTokenUtil.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/819ed402a13d4fc3985c2b8fc6da362c956d8308/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2Fjwt%2FJwtTokenUtil.java)|4.93%|:x:|

|Total Project Coverage|56.51%|:green_apple:|
|:-|:-:|:-:|

**Review by Ewentualnie — APPROVED** [review]

**Pull request merged**